server/cmwell-ws/app/actions/DashBoard.scala

```diff
@@ -274,19 +271,6 @@ class DashBoard @Inject()(crudServiceFS: CRUDServiceFS)(implicit ec: ExecutionCo
     @volatile private[this] var detail: DetailedHealthData = Map.empty[String,ColorTuple4]
     @volatile private[this] var detailTime = new DateTime(0L)
 
-    def init: Future[(DetailedHealthData,TlogStatus)] = {
```

Comment:
again, no context here, but making sure we really don't need the part of `init` that takes care of detailed-health